pom.xml

```diff
@@ -51,6 +51,11 @@
             <url>http://repository.jboss.org/licenses/apache-2.0.txt</url>
             <distribution>repo</distribution>
         </license>
+        <license>
+            <name>Eclipse Public License - v 1.0</name>
+            <url>https://www.eclipse.org/legal/epl-v10.html</url>
+            <distribution>repo</distribution>
+        </license>
     </licenses>
```

Comment:
I think the licenses on the main pom.xml file are the wildfly-core license and not an aggregation of the licenses used by the modules shipped with it. "Eclipse Public License" should be removed from the main pom.xml

Reply:
Sorry, I should have read the Jira description first. It seems pretty clear we need it here, so, ignore my comment

Reply:
It's not clear to me, so I've put the 'hold' label on this while I get expert advice. :)

Reply:
@RanabirChakraborty The feedback I've gotten is this is not required, so please remove it. Thanks for your patience.

Reply:
Thanks! @bstansberry did the changes :)